cmake/modules/FindWine.cmake

```diff
@@ -24,6 +31,10 @@ STRING(REPLACE " " ";" WINEBUILD_FLAGS "${WINEBUILD_OUTPUT}")
 
 FOREACH(FLAG ${WINEBUILD_FLAGS})
 	IF("${FLAG}" MATCHES "libwinecrt0.a.*")
+		STRING(REGEX REPLACE "/wine/libwinecrt0.a.*" "" FLAG "${FLAG}")
+
+		SET(WINE_64_LIBRARY_DIR "${FLAG}/")
```

Comment:
Unusual, but it is possible to use 32-bit WINE toolchain `wine32-tools` on 64-bit Debian/Ubuntu. In that case, 64-bit VST compilation needs the inverse replacing(32-bit toolchain uses 32-bit libraries, even if `-m64` is specified).

Reply:
Is that something we have supported before and that this PR breaks?